src/basics.md

```diff
@@ -168,7 +158,7 @@ fn main() {
 }
 ```
 
-Alternatively, one can use the [`Range`](https://doc.rust-lang.org/rand/rand/distributions/range/struct.Range.html) distribution.
+Alternatively, one can use the [`Range`] distribution (otherwise known as uniform).
```

Comment:
I would rename the thing if I could. I may actually try, but it's not high up the priorities of what that crate needs.